.github/workflows/workflow.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+env:
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+  DOCKER_TLS_CERTDIR: ''
+
+services:
+  docker:dind:
+    image: docker:dind
+    entrypoint: ['env', '-u', 'DOCKER_HOST']
+    command: ['dockerd-entrypoint.sh']
+
+jobs:
+  download-vectors:
+    runs-on: ubuntu-latest
+    steps:
+     - run: |
+        wget -q https://github.com/explosion/sense2vec/releases/download/v1.0.0/s2v_reddit_2015_md.tar.gz
+        tar -xzf s2v_reddit_2015_md.tar.gz -C src
+        rm s2v_reddit_2015_md.tar.gz
+    uses: actions/upload-artifact@v2
+    with:
+      path: src/s2v_old
+    
+  test-server:
+    runs-on: ubuntu-latest
+    needs: [download-vectors]
+    container: docker/compose
```

Comment:
Remove the `container` line from each job. For jobs requiring Docker Compose, nothing is required since it's built-in to GitHub Actions. For others such as Node.js, use something like the Node.js GitHub Actions to pull in the latest LTS.

Reply:
I am not sure how to use node actions, please review the updated file